.buildkite/commands/run-ui-tests.sh

```diff
@@ -40,8 +40,9 @@ echo "--- 📦 Zipping test results"
 cd build/results/ && zip -rq JetpackUITests.xcresult.zip JetpackUITests.xcresult && cd -
 
 echo "--- 💥 Collecting Crash reports"
+mkdir -p build/results/crashes
 find ~/Library/Logs/DiagnosticReports -name '*.ips' -print0 | while read -d $'\0' -r file; do
-  cp "$file" "build/results/[CRASH]$(basename "$file")"
+  cp "$file" "build/results/crashes/$(basename "$file")"
 done
```

Comment:
True! 

> Which means we would even directly use the -exec primitive of find now that the command is simpler

I believe it worth fixing it because it reads much better and I'm not committed to bad past choices. =P

I've put [this PR up](https://github.com/wordpress-mobile/WordPress-iOS/pull/20658).